addons/website_slides/static/src/xml/slide_quiz_create.xml

```diff
@@ -34,7 +36,7 @@
 
     <t t-name="slide.quiz.answer.line">
         <div class="o_wslides_js_quiz_answer row align-items-center mb-1" t-attf-data-answer-id="#{answer ? answer.id : ''}" >
-            <div class="col offset-1">
+            <div class="col offset-1 o_wslides_quiz_form_answer">
```

Comment:
Actually, I tested removing `offset-1 o_wslides_quiz_form_answer` in favor of a simple `ml-3` and it seems like it's behaving correctly on both standard and mobile views. Can you try it?